xyz.parlatype.Parlatype.json

```diff
@@ -0,0 +1,76 @@
+{
+    "app-id": "xyz.parlatype.Parlatype",
+
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "46",
+    "sdk": "org.gnome.Sdk",
+
+    "command": "parlatype",
+
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--filesystem=home:ro",
+        "--talk-name=org.freedesktop.secrets",
+        "--own-name=org.mpris.MediaPlayer2.parlatype"
```

Comment:
I'm pretty sure you need `org.mpris.MediaPlayer2.parlatype.*`

Reply:
Done in https://github.com/flathub/flathub/pull/5312/commits/e2be8e385929578ce11968166683c564b88bfc73. Though it worked for me before, now too. I was like in org.gnome.Rhythmbox3 before.